src/app/view/components/_common/iframe/iframe-component/iframe.component.ts

```diff
@@ -58,20 +61,22 @@ export class IframeComponent implements OnInit {
      * @memberof IframeComponent
      */
     checkSessionExpired(): void {
-        if (this.iframeElement && this.iframeElement.nativeElement.contentWindow) {
-            const currentPath = this.iframeElement.nativeElement.contentWindow.location.pathname;
-
-            if (currentPath.indexOf('/c/portal_public/login') !== -1) {
-                this.loginService.logOutUser().subscribe(
-                    (_data) => {},
-                    (error) => {
-                        this.loggerService.error(error);
-                    }
-                );
-            }
+        if (!!this.getIframeWindow() && this.getIframeLocation().pathname.indexOf('/c/portal_public/login') !== -1) {
```

Comment:
maybe we shoud create a know url constant storage somewhere for this ''/c/portal_public/login''

I dont know something like

KNOW_URL = {
    LOGIN: '/c/portal_public/login'
}

Reply:
It might be cool, but maybe in some of the refactors.